src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -312,51 +376,21 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
-    uut_ppa_zone_data = self.triggerCpasAndRetrievePpaZone(
-        ppa_id,
+    # PPA Zone record.
+    uut_ppa_zone_data = self.triggerFadAndRetrievePpaZone(
         ssl_cert=config['sasTestHarnessCert'],
         ssl_key=config['sasTestHarnessKey'])
 
     # Configure the Census Tract Directory that the PPA uses
     ppa.ConfigureCensusTractDriver(DEFAULT_CENSUSTRACTS_DATAPATH)
 
-    # PpaCreationModel requires the input registrationRequests to have 'installationParam'.
-    # But this parameter is removed for devices where conditionals are pre-loaded.
-    # Adding the 'installationParam' into registrationRequests by taking the
-    # corresponding values from conditionalRegistrationData.
-    for device in config['registrationRequests']:
-      if not device.has_key('installationParam'):
-        for conditional_params in config['conditionalRegistrationData']:
-          # Check if FCC_ID+Serial_Number present in registrationRequest
-          # and conditional_params match and add the 'installationParam'.
-          if conditional_params['fccId'] == device['fccId'] and \
-                          conditional_params['cbsdSerialNumber'] == device['cbsdSerialNumber']:
-            # The following REG-conditional parameters are required to present in
-            # installationParam Object and cbsdCategory in RegistrationRequest for PPA
-            # reference model to determine PPA contour boundary.
-            install_params = {}
-            install_params['eirpCapability'] = conditional_params['installationParam'][
-                'eirpCapability']
-            install_params['antennaAzimuth'] = conditional_params['installationParam'][
-                'antennaAzimuth']
-            install_params['longitude'] = conditional_params['installationParam'][
-                'longitude']
-            install_params['latitude'] = conditional_params['installationParam'][
-                'latitude']
-            install_params['antennaGain'] = conditional_params['installationParam'][
-                'antennaGain']
-            install_params['indoorDeployment'] = conditional_params['installationParam'][
-                'indoorDeployment']
-            install_params['antennaBeamwidth'] = conditional_params['installationParam'][
-                'antennaBeamwidth']
-            install_params['height'] = conditional_params['installationParam']['height']
-            device.update({
-                'installationParam': install_params
-            })
-            device.update({
-                'cbsdCategory': conditional_params['cbsdCategory']
-            })
+    # Asserts the REG-Conditional value doesn't exist in the registrationRequest,
+    # it required to be exist in the registrationRequest data
+    try:
+      self.assertRegConditionalsForPpaRefModel(config['registrationRequests'],
+                                               config['conditionalRegistrationData'])
+    except Exception as err:
+      self.fail(err.message)
```

Comment:
Not needed.

Reply:
Debug   log was added for testing purpose has been removed now